### DISCUSSION TITLE
Error: P1001 : Can't reach database server at 'aws.connect.psdb.cloud':'3306'

### DISCUSSION THREAD

**Discussion by Katlego-K** [prisma/prisma#22490] [Q&A]
### Question

Good evening everybody, I'm getting the above mentioned error when I try to run: npx prisma db push on my Windows machine.
Below is the full error details:
Environment variables loaded from .env
Prisma schema loaded from prisma\schema.prisma
Datasource "db": MySQL database "ekat" at "aws.connect.psdb.cloud"
![2023-12-21_19h02_41](https://github.com/prisma/prisma/assets/34712841/36674bd1-1de4-4a8e-a71e-647fc5ce0e53)
![2023-12-21_19h02_41](https://github.com/prisma/prisma/assets/34712841/873519b4-304c-4860-96c9-335b1a0168a0)

Error: P1001: Can't reach database server at `aws.connect.psdb.cloud`:`3306`

Please make sure your database server is running at `aws.connect.psdb.cloud`:`3306`.

Please assist and I have attached the screenshot. Thank you in advanced

### Expected behavior (optional)

To be connected successfully

### Information about Prisma Schema, Client Queries and Environment (optional)

```prisma
// Add your schema.prisma
```

generator client {
  provider = "prisma-client-js"
}

datasource db {
  provider = "mysql"
  url      = env("DATABASE_URL")
  relationMode = "prisma"
}

model User {
  id Int @id @default(autoincrement())
  name String? @db.VarChar(255)
  email String @unique @db.VarChar(255)
  password String @db.VarChar(255)
}
// Add any relevant Prisma Client queries here
```

- OS: <!--[e.g. macOS, Windows, Debian, CentOS, ...]-->
- Database: <!--[PostgreSQL, MySQL, PlanetScale, MariaDB, SQL Server, SQLite, MongoDB or CockroachDB]-->
- Node.js version: <!--[Run `node -v` to see your Node.js version]-->

- Run `prisma -v` to see your Prisma version and paste it

**Comment by ludralph** [comment]
Hi @Katlego-K 👋 

Thank you for raising this question.

Can you connect to the Database using a GUI tool like [DataGrip](https://www.jetbrains.com/datagrip/) or [TablePlus](https://tableplus.com/)? Also, how did you define the Database URL in the `.env` file?

**Reply by ludralph** [reply]
Hey @Katlego-K ,

It would be really helpful if you could provide the additional information that we had asked about last time. If your problem is resolved by now or somehow doesn't feel relevant any more, you can simply close the Discussion as well.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
